### ISSUE TITLE
After fine-tuning pre-trained WideResNet, the ID classification drops a lot?

### ISSUE THREAD

**Issue by lygjwy** [hendrycks/outlier-exposure#20]
Hello! Thank you for your excellent work. I have some questions about the implementation.
When I change the train_loader_ood with shuffle = True and the ID classification drops a lot after finetuning the pre-trained WideResNet with lr 0.001 and weight_decay 0.0005. The fine-tuned WideResNet in Cifar10 has 88.59% test classification accuracy compared with 94.39%.
Instead, you use a random offset to induce the randomness. Could you please clarify the reason for this operation? Thanks again!

**Comment by QingyangZhang** [comment]
I am curious about this, too.
